pkg/models/resources/v1alpha3/persistentvolume/persistentvolume.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2019 The KubeSphere Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolume
+
+import (
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/informers"
+
+	"kubesphere.io/kubesphere/pkg/api"
+	"kubesphere.io/kubesphere/pkg/apiserver/query"
+	"kubesphere.io/kubesphere/pkg/models/resources/v1alpha3"
+)
+
+const (
+	storageClassName = "storageClassName"
+)
+
+type persistentVolumeGetter struct {
+	informers informers.SharedInformerFactory
+}
+
+func New(informer informers.SharedInformerFactory) v1alpha3.Interface {
+	return &persistentVolumeGetter{informers: informer}
+}
+
+func (p *persistentVolumeGetter) Get(namespace, name string) (runtime.Object, error) {
+	pv, err := p.informers.Core().V1().PersistentVolumes().Lister().Get(name)
+	if err != nil {
+		return pv, err
+	}
+	return pv, nil
```

Comment:
You can just return `pv, err` here regardless the `err` is `nil` or not

Reply:
Okay